packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -149,9 +149,6 @@
     .paragraph ul:not(.perseus-widget-radio) {
         .legacy-typography;
         padding-left: 35px;
-    }
-
-    .paragraph ul:not(.perseus-widget-radio) li {
         list-style-type: disc;
```

Comment:
Moving this made me nervous but I can't imagine a scenario where this would be different unless we were overriding some weird classes replacing bullets with emojis at the list item level somewhere. In that case, we should probably get rid of whatever that class is.

Reply:
Makes sense to me